src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
ahh ok. So there is a bit of a conflict inherent in that, since some also would rather that the days not shrink below a certain size and scroll beyond that. We should make sure then we have css defaults that are easy to adjust/override for folks that want either behavior.

In your case getting everything to fit on a page should be a simple matter of setting the calendar height to the page size and the flexbox styles could handle everything from there for all the inner components. as it is that won't work without some adjustments up the line... in any case I can tweak that stuff later, nothing to be done here now.